test/test_datapipe.py

```diff
@@ -2722,6 +2722,45 @@ def construct_sharded_pipe():
         with self.assertRaises(Exception):
             dp.apply_sharding(2, 1, sharding_group=SHARDING_PRIORITIES.DEFAULT)
 
+    # Test tud.datapipes.iter.grouping.SHARDING_PRIORITIES for backward compatbility
+    # TODO: Remove this test once tud.datapipes.iter.grouping.SHARDING_PRIORITIES is deprecated
+    def test_sharding_groups_in_legacy_grouping_package(self):
+        from torch.utils.data.datapipes.iter.grouping import SHARDING_PRIORITIES as LEGACY_SHARDING_PRIORITIES
```

Comment:
Should we add a `assertWarning` at import?
And, other tests can be removed as they just references to the right classes.

Reply:
> And, other tests can be removed as they just references to the right classes.

discussed offline, just use them as some smoke integration test~